pom.xml

```diff
@@ -950,17 +949,11 @@
       <activation>
         <jdk>[1.9,)</jdk>
       </activation>
-      <build>
-        <plugins>
-          <plugin>
-            <artifactId>maven-compiler-plugin</artifactId>
-            <configuration>
-              <release>8</release>
-              <testRelease>8</testRelease>
-            </configuration>
-          </plugin>
-        </plugins>
-      </build>
+      <properties>
+        <maven.compiler.release>8</maven.compiler.release>
+        <maven.compiler.testRelease>8</maven.compiler.testRelease>
+        <animal.sniffer.skip>true</animal.sniffer.skip>
```

Comment:
Technically a change in behavior, but `release` serves the same purpose as Animal Sniffer.

Reply:
Well, almost. With A.S. you can use `@IgnoreJRERequirements` on a particular class/method. I do not recommend it (better to use reflection) and would not miss that capability being dropped.